java/src/jmri/jmrix/acela/serialdriver/configurexml/ConnectionConfigXml.java

```diff
@@ -326,7 +326,8 @@ protected void unpackElement(Element shared, Element perNode) {
             tc.initializeAcelaNode(node);
         }
         // Do not let the Acela network poll until we are really ready
-        ((AcelaTrafficController)adapter).setReallyReadyToPoll(true);
+
+        ((AcelaSystemConnectionMemo) adapter.getSystemConnectionMemo()).getTrafficController().setReallyReadyToPoll(true);
     }
```

Comment:
This is actually a really good example of why Casts Are Considered Harmful.  We only discovered late, at user run time, what FindBugs had been saying:  There was no reason to believe that cast in the original line was going to work.

Incidentally, it would be good to remove the `(AcelaSystemConnectionMemo)` cast in the replacement line, along with the next one. Those are both redundant.